src/api/session/closeSession.ts

```diff
@@ -0,0 +1,57 @@
+import { ZodError } from 'zod';
+import {
+  ApiErrorResponse,
+  CloseSessionPayloadType,
+  CloseSessionPayloadCallType,
+  RemoveBasicAuthenticationPayloadType
+} from '../../types';
+import { sdkApiError, fetchWithTimeout, getHeaders } from '../../utils';
+
+/**
+ * Unassign an alias from a PeerId.
+ *
+ * @param apiEndpoint - The API endpoint
+ * @param apiToken - The API token used to authenticate the request.
+ * @param body - The payload containing the details of the alias to remove.
+ * @returns A Promise that resolves to true if the alias was successfully removed.
+ * @throws An error that occurred while processing the request.
+ */
+export const closeSession = async (
+  payload: CloseSessionPayloadType
+): Promise<boolean> => {
+  const { protocol, listeningIp, port, apiToken, apiEndpoint, } = payload;
+  const url = new URL(`api/v3/session/${protocol}/${listeningIp}/${port}`, apiEndpoint);
+  const rawResponse = await fetchWithTimeout(
+    url,
+    {
+      method: 'DELETE',
+      headers: getHeaders(apiToken),
+    },
+    payload.timeout
+  );
+
+  // received expected response
+  if (rawResponse.status === 204) {
```

Comment:
Do you really care for the 204 here? Wouldn't `rawResponse.ok` be the better check?

Reply:
Changed